### PR TITLE
#IM add recipe for hoggorm (explorative multivariate statistics)

### PR DESCRIPTION
**Pull request by andife**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/hoggorm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - sphinx
+    - numpy >=1.11.3
```
Comment (scopatz) [inline recipes/hoggorm/meta.yaml:23]: You probably don't need numpy to build the package

**Comment by scopatz** [comment]
This is a noarch python package so it only needs to pass on Linux

File: recipes/hoggorm/meta.yaml (hunk 1 above)
Comment (andife) [reply recipes/hoggorm/meta.yaml:23]: Thanks for reviewing.
I don't understand that as it's not included in the build-section. Removing it from the host section and the 'staged-recipes (linux_64 linux)' will fail.

Comment (scopatz) [reply recipes/hoggorm/meta.yaml:23]: Are you compiling and linking against numpy?

Comment (andife) [reply recipes/hoggorm/meta.yaml]: Now, I think understand your point. I've found https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy and also a discussion at staged-recipes. I do not exactly know if it is needed as build-req.
grep -H numpy * shows
README.rst:- numpy >= 1.9
requirements.txt:numpy >= 1.11.3
setup.cfg:	numpy>=1.9

I removed the dependency

Comment (andife) [reply recipes/hoggorm/meta.yaml]: According to a comment in https://gitter.im/conda-forge/conda-forge.github.io
"Normally it shouldn't be but it looks like the package is importing itself when running setup.py to get the version. It's generally seen as a bad practice and therefore uncommon but in this case you will need numpy in host. It's worth including a comment in the meta.yaml explaining why it's there."

File: README.md [hunk 2]
```diff
@@ -131,11 +131,11 @@ If your PR is passing all checks, but has not been acted on by the staged recipe
 maintainers, you can ping @conda-forge/staged-recipes to request action. You do
 not need to wait any specific amount of time once the recipe is ready to go.
 
-If your recipe still does not recieve any attention after a few days, you may
+If your recipe still does not receive any attention after a few days, you may
```
Comment (scopatz) [inline README.md:134]: Please do not modify files in a recipe PR outside of the recipes dir.  Please make a separate PR for this file, if you want. Thanks!

Comment (andife) [reply README.md]: Thank you for the hint. I reverted the spelling commit. Now the recipe should be fine.

**Comment by scopatz** [comment]
Thanks!
